### PR TITLE
Fixes an issue on cleanup with mapped directories

### PR DESCRIPTION
**Pull request by jterry75**
Resolves: #131

### CODE REVIEW

**Comment by jterry75** [comment]
@jhowardmsft - Has verified the fix

**Comment by jhowardmsft** [comment]
LGTM

File: service/gcs/core/gcs/cleanup.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ func (c *gcsCore) cleanupContainer(containerEntry *containerCacheEntry) error {
 	}
 
 	diskMap := containerEntry.MappedVirtualDisks
-	disks := make([]prot.MappedVirtualDisk, 0, len(diskMap))
```
Comment (beweedon) [inline service/gcs/core/gcs/cleanup.go]: Correct me if I'm wrong, but if we use make like this, doesn't it create a slice of length `len(diskMap)` full of default elements, and then `append` just adds on to this slice? So we'd end up with a slice half full of default `MappedVirtualDisk`s?

Comment (beweedon) [reply service/gcs/core/gcs/cleanup.go]: You could also do it by not specifying the capacity as your have, and assigning the disk/dir at the correct index in the assignment loop as well. I honestly like that better I think :)

Comment (jterry75) [reply service/gcs/core/gcs/cleanup.go]: Oh good call.

**Pull request merged**